src/views/Setup.vue

```diff
@@ -28,13 +28,13 @@
           <p class="progress-subtitle">
             {{ format(percentage) }}
             <DotsLoading
-              v-if="!synced && theme === THEMES.DARK"
+              v-if="theme === THEMES.DARK"
```

Comment:
We could create `color` computed property and move the `if else` logic there to simplify the HTML